index.html

```diff
@@ -118,7 +114,140 @@ <h1 class="fs-40 fw-7 introduction">
+              </li>
+              <li>
+                <input
+                  type="button"
+                  class="btn-details txt5 fw-5 fs-17"
+                  value="See Live"
+                />
+              </li>
+              <li class="desktop-only">
+                <input
+                  type="button"
+                  class="btn-details txt5 fw-5 fs-17"
+                  value="See More"
+                />
+              </li>
+            </ul>
+          </div>
+        </div>
+        <div class="section-workproject bg-white">
+          <div class="working-container">
+            <div class="project-img1">
+              <img
+                src="./assets/images/mobile/workimg2.svg"
+                alt="project one"
+                class="work-img"
+                id="work-img2"
+              />
+            </div>
+            <ul class="work-project">
+              <li class="fswctitle">Tonic</li>
+              <li class="desktop-only">&#215;</li>
+            </ul>
+            <ul class="project-techn">
+              <li class="fw-6 fs-13 txtd">CANOPY</li>
+              <li class="fw-6 fs-13 techn-list">
+                <div class="dot-icon"></div>
+                Back End Dev
+              </li>
+              <li class="fw-6 fs-13 techn-list">
+                <div class="dot-icon"></div>
+                2023
+              </li>
+            </ul>
+            <p class="project-description">
+              Lorem ipsum dolor sit amet consectetur adipisicing elit. Officia
+              quo fuga quisquam sit porro officiis cumque totam ipsum similique.
+              Quasi dignissimos voluptas ex nostrum soluta repudiandae suscipit
+              delectus aliquid labore?
+            </p>
+            <ul class="project-language">
+              <li class="lang-list">
+                <a class="fw-5 fs-12 txt2 lang-link">Html</a>
+              </li>
+              <li class="lang-list desktop-only">
+                <a class="fw-5 fs-12 txt2 lang-link">Ruby on Rails</a>
+              </li>
+              <li><a class="fw-5 fs-12 txt2 lang-link">Css</a></li>
+              <li><a class="fw-5 fs-12 txt2 lang-link">Javascript</a></li>
+            </ul>
+            <ul class="project-details">
+              <li class="desktop-only">
+                <input
+                  type="button"
+                  class="btn-details txt5 fw-5 fs-17"
+                  value="see Source"
+                />
+              </li>
+              <li>
+                <input
+                  type="button"
+                  class="btn-details txt5 fw-5 fs-17"
+                  value="See Live"
+                />
+              </li>
+              <li class="desktop-only">
+                <input
+                  type="button"
+                  class="btn-details txt5 fw-5 fs-17"
+                  value="See More"
+                />
+              </li>
+            </ul>
+          </div>
```

Comment:
- [ ] Great job so far :muscle: However I noticed that you didn't implement the requirements that the previous reviewer asked of you.  According to the [project requirements](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/8_popup.md), You need to use a `JavaScript array` to store all of the information for all projects. :100: 
- [ ] For each project you need to store the following pieces of data in a JavaScript object, at a minimum: name, description, featured image, technologies, link to live version, link to source.
- [ ] Just as he mentioned, kindly remove this entire section and ensure that the projects section is created dynamically with the information from the `Javascript array `  you create (all the HTML of the section is created when the page loads). :100: This section should be added via javascript :+1: 
- [ ] After this, you should implement popups for both mobile and desktop screen sizes as shown in the Figma Design
- Mobile Popup
![image](https://user-images.githubusercontent.com/102699958/224831696-9c0b4551-aaee-4034-b4df-5d0f7da936eb.png)

- Desktop Popup
![image](https://user-images.githubusercontent.com/102699958/224831557-7b480048-b077-4dbf-8872-58bbe08dcfc6.png)
- [ ] Lastly, as the previous reviewer mentioned, please refer to the [project requirements](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/8_popup.md) and ensure that you have completed all the requirements because currently, your project is missing all of the project requirements for this milestone :+1: